include/h2o.h

```diff
@@ -1037,7 +1039,10 @@ struct st_h2o_req_t {
 typedef struct st_h2o_accept_ctx_t {
     h2o_context_t *ctx;
     h2o_hostconf_t **hosts;
-    SSL_CTX *ssl_ctx;
+    struct {
+        SSL_CTX *ssl_ctx;
+        h2o_iovec_t *http2_origin_frame;
+    } ssl;
```

Comment:
Let's not move `ssl_ctx` into a new field, but have both `ssl_ctx` and `http2_origin_frame` direct members of `h2o_accept_ctx_t`.

The reasons are: 1. the change would be smaller 2. in terms of protocol, origin frames can be sent even when SSL is not used.